src/Lean/Meta/CompletionName.lean

```diff
@@ -0,0 +1,47 @@
+/-
+Copyright (c) 2023 Lean FRO. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Leonardo de Moura
+-/
+
+import Lean.Meta.Basic
+import Lean.Meta.Match.MatcherInfo
+
+/-!
+
+This exports a predicate for checking whether a name should be made visible
+in auto-completion and other tactics.  It uses a blacklist environment extension
+to enable names in an environment to be specifically hidden.
+-/
+namespace Lean.Server.Completion
+
+builtin_initialize completionBlackListExt : TagDeclarationExtension ← mkTagDeclarationExtension
+
+end Lean.Server.Completion
```

Comment:
I have to keep this extension in `Lean.Server.Completion` for tests to pass, but I thought that through an educated guess.  How could one move the namespace of this extension?